entoas/internal/oastypes/entc.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019-present Facebook
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build ignore
+// +build ignore
+
+package main
+
+import (
+	"log"
+
+	"entgo.io/contrib/entoas"
+	"entgo.io/ent/entc"
+	"entgo.io/ent/entc/gen"
+)
+
+func main() {
+	ex, err := entoas.NewExtension()
+	if err != nil {
+		log.Fatalf("creating entoas extension: %v", err)
+	}
+	err = entc.Generate("./schema", &gen.Config{
+		Templates: []*gen.Template{gen.MustParse(
+			gen.NewTemplate("").Parse(`{{ define "model/stringer" }}// No String() implementation{{ end }}`),
+		)},
```

Comment:
I'm afraid users will use it as a reference example (and copy/paste this). Let's remove this template and rename the `string` field in ent/schema. WDYT?

Reply:
Agreed.